extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamInfoItemExtractor.java

```diff
@@ -58,7 +58,7 @@ public interface StreamInfoItemExtractor extends InfoItemExtractor {
     /**
      * Parses the number of views
      *
-     * @return the number of views or -1 for live streams
+     * @return the number of views or -1 if not available
```

Comment:
Maybe
```
     * Parses the number of views (or of current viewers in case of live streams)
     *
     * @return the number of views (or of current viewers in case of live streams) or -1 if not available
```